docs/Data_Portal/Users_Guide/quick_start.md

```diff
@@ -18,10 +18,10 @@ On the upper-left is the GDC Data Portal logo, which links to the home page of t
 On the right are the following features:
 * **Video Guides**: links to videos demonstrating the various features of the Data Portal
 * **Send Feedback**: provides information on how to send feedback to our team
-* **Browse Annotations**: links to the Annotations Browser, where the user can view and search for annotations that may be of use when analyzing GDC data
+* **Browse Annotations**: links to the Annotations Browser, where the user can view and search for [annotations](/Encyclopedia/pages/Annotations/) that may be of use when analyzing GDC data
 * **Manage Sets**: allows the user to review gene and mutation sets that have been saved, upload new sets, and delete existing sets
 * **Cart**: where data files of interest can be added for download
-* **Login**: allows authentication for access to controlled access datasets
+* **Login**: allows authentication for access to [controlled access datasets](/Encyclopedia/pages/Controlled_Access/). Once authentication is successful, the eRA Commons username will be displayed in place of the "Login" button. Clicking on the username will then allow users to see which projects they have access to and to download an [authentication token](/Data/Data_Security/Data_Security/#authentication-tokens) for use with the [GDC Data Transfer Tool](/Data_Transfer_Tool/Users_Guide/Getting_Started/) and the [API](/API/Users_Guide/Getting_Started/).
```

Comment:
please check that I've done the links correctly.

Reply:
also lemme know if we need more on access, etc.

Reply:
Same as above - I think these needs to be: 
(/Encyclopedia/pages/Controlled_Access.md)
(/Data/Data_Security/Data_Security.md/#authentication-tokens)
(/Data_Transfer_Tool/Users_Guide/Getting_Started.md)
(/API/Users_Guide/Getting_Started.md)

@wwysoc2 can you verify?

Reply:
I see some existing links with .md and some without. 😵 But I think I'll add them to be safe once bill confirms. Thanks for catching!